### ISSUE TITLE
Puppet livery should work on RHEL machines

### ISSUE THREAD

**Issue by rtyler** [rtyler/blimpy#62]
```
which: no puppet in (/usr/lib64/qt-3.3/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin)
puppet.sh                                                                                                                                                                                                                                         100%  986     1.0KB/s   00:00    
gempath.sh                                                                                                                                                                                                                                        100%   61     0.1KB/s   00:00    
++ uname
+ OSNAME=Linux
+ '[' Linux = FreeBSD ']'
+ export PATH=/var/lib/gems/1.8/bin:/usr/local/bin:/usr/lib64/qt-3.3/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+ PATH=/var/lib/gems/1.8/bin:/usr/local/bin:/usr/lib64/qt-3.3/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+ which puppet
which: no puppet in (/var/lib/gems/1.8/bin:/usr/local/bin:/usr/lib64/qt-3.3/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin)
+ '[' 1 -ne 0 ']'
+ apt-get update
./puppet.sh: line 32: apt-get: command not found
+ apt-get install -y ruby1.8 ruby1.8-dev libopenssl-ruby1.8 rubygems
./puppet.sh: line 34: apt-get: command not found
+ gem install puppet --version '~> 2.7' --no-ri --no-rdoc
./puppet.sh: line 39: gem: command not found
./gempath.sh: line 5: exec: puppet: not found

```

**Comment by jblaine** [comment]
https://github.com/rtyler/blimpy/pull/64

**Comment by rtyler** [comment]
Fixed in 0.6.5 which was just released
